tests/unit/algos/list/delimiterMaster.spec.js

```diff
@@ -59,6 +145,61 @@ function getListingKey(key, vFormat) {
             assert.strictEqual(delimiter.skipping(), SKIP_NONE);
         });
 
+        it('should list all subprefixes when handleStreak is applied as in RepdServer', () => {
+            const prefix = '_Common-Prefix/';
+            const delimiter = new DelimiterMaster({
+                prefix,
+                delimiter: '/',
+                startAfter: '',
+                continuationToken: '',
+                v2: true,
+                fetchOwner: false }, fakeLogger, vFormat);
+            
+            [...generateKeys(1000)].forEach((ob, idx) => {
+                const {key, isDeleteMarker} = ob;
+
+                // simulate not doing a raw listing when key < params.gte
+                // this represents a key not reaching the read stream from leveldb
+                if (params.gte && key < params.gte) {
+                    throw new Error('Invariant violated. Keys may be missing from listing.');
+                }
+
+                if (key.indexOf(VID_SEP) === -1) { // master key
```

Comment:
Instead of `indexOf` you can do includes: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes